common/src/main/java/com/wynntils/core/features/overlays/Overlay.java

```diff
@@ -175,4 +217,15 @@ public HorizontalAlignment getRenderHorizontalAlignment() {
     public VerticalAlignment getRenderVerticalAlignment() {
         return verticalAlignmentOverride == null ? position.getVerticalAlignment() : verticalAlignmentOverride;
     }
+
+    public Map<Corner, Vec2> getCornersMap() {
```

Comment:
If I get this right you put the four corners in a map, and then you extract two of them in Edge. This is the only use of this method, yes?

What if you make it into a switch expression `getCornerPos` that takes in a Corner and return the position for that corner?

Reply:
Ah, I see now, you also iterate over it in the manager. But you could probably still have this a getter for a method, and iterate over all possible values of the Corner enum there.